website/views/options/inventory/time-travelers.jade

```diff
@@ -29,7 +29,7 @@
                 popover-placement='right', popover-append-to-body='true',
                 ng-click='clickTimeTravelItem("#{type}",item)')
       li.customize-menu.inventory-gear
-        menu.pets-menu(label='{{::set.text}}', ng-repeat='set in Content.timeTravelerStore(user.items.gear.owned)')
+        menu.pets-menu(label=env.t('{{::set.text}}'), ng-repeat='set in Content.timeTravelerStore(user.items.gear.owned)')
```

Comment:
If you set the translation function in the content object, then all you need to do is: `label='{{::set.text()}}'`